docker-compose-dev.yml

```diff
@@ -0,0 +1,18 @@
+db:
+  image: mysql:5.7
```

Comment:
another version of mysql? Why not 5.5 or 5.6 which are tested on travis?

Reply:
Looks like it's time to change travis because 5.7 is what we specify in out [production-ready docker-compose](https://github.com/allegro/ralph/blob/ng/contrib/docker-compose.yml.tmpl) in contrib/ :)

Reply:
done